config/owasp/suppressions.xml

```diff
@@ -95,26 +95,26 @@
   </suppress>
   <suppress>
     <notes><![CDATA[
-   file name: spring-cloud-azure-starter-active-directory-4.0.0.jar
+   file name: spring-cloud-azure-starter-active-directory-4.4.0.jar
```

Comment:
If you've not already, could you have a quick look to see if any of the above can be removed now that it's upgraded?

Reply:
Good point. Have removed the ones we no longer need :)